apps/emqx_dashboard_sso/src/emqx_dashboard_sso_api.erl

```diff
@@ -280,7 +280,7 @@ handle_backend_update_result({error, Reason}, _) ->
 
 to_json(Data) ->
     emqx_utils_maps:jsonable_map(
-        Data,
+        emqx_utils:redact(Data),
```

Comment:
Nit: mb. rename the fun to `to_redacted_json` to avoid misuse 🤔

Reply:
✅